### PR TITLE
add better default example for jsonschema

### PR DESCRIPTION
**Pull request by inkhey**
related to #4119 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/examples/user_custom_properties/default/schema.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "",
+  "description": "Info about this user",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/schema.json:4]: I think we can leave the description field empty.

File: backend/examples/user_custom_properties/default/locale/fr.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "Info about this user": "Info sur l'utilisateur",
+  "Tagline": "Slogan",
+  "Localization": "Localisation",
+  "Hobbies": "Loisir",
+  "Website": "Site web",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/fr.json:6]:
```suggestion
  "Personal Website": "Site web personnel",
```

File: backend/examples/user_custom_properties/default/schema.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    "hobbies": {
+      "title": "Hobbies",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "website": {
+      "type": "string",
+      "title": "Website"
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/schema.json:25]:
```suggestion
      "title": "Personal Website"
```

File: backend/examples/user_custom_properties/default/schema.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    "birthday": {
+      "type": "string",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/schema.json:28]:
```suggestion
      "type": "string",
      "format": "date"
```

File: backend/examples/user_custom_properties/default/ui.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  "hobbies":{
+    "tracim:display_group": "public_profile_first"
+  },
+  "website":{
+    "tracim:display_group": "public_profile_first"
+  },
+  "birthday": {
+    "ui:widget": "alt-date",
+    "tracim:display_group": "public_profile_first"
+  },
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/ui.json:17]:
```suggestion
  "birthday": {
    "tracim:display_group": "public_profile_first"
  },
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: backend/examples/user_custom_properties/default/locale/pt.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
+  "Info about this user": "",
+  "Tagline": "",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/pt.json:3]:
```suggestion
  "Tagline": "Slogan",
```

File: backend/examples/user_custom_properties/default/locale/pt.json [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  "Localization": "",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/pt.json:4]:
```suggestion
  "Localization": "Localização",
```

File: backend/examples/user_custom_properties/default/locale/pt.json [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+  "Hobbies": "",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/pt.json:5]:
```suggestion
  "Hobbies": "Passatempos",
```

File: backend/examples/user_custom_properties/default/locale/pt.json [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+  "Personal Website": "",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/pt.json:6]:
```suggestion
  "Personal Website": "Website pessoal",
```

File: backend/examples/user_custom_properties/default/locale/pt.json [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+  "Birthday": "",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/pt.json:7]:
```suggestion
  "Birthday": "Data de nascimento",
```

File: backend/examples/user_custom_properties/default/locale/pt.json [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+  "Telephone": "",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/pt.json:8]:
```suggestion
  "Telephone": "Telefone",
```

File: backend/examples/user_custom_properties/default/locale/pt.json [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+  "Languages": "A CHANGER",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/pt.json:9]:
```suggestion
  "Languages": "Línguas",
```

File: backend/examples/user_custom_properties/default/locale/pt.json [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+  "Personal Page": ""
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/pt.json:10]:
```suggestion
  "Personal Page": "Página pessoal"
```

File: backend/examples/user_custom_properties/default/locale/pt.json [hunk 14]
```diff
@@ -0,0 +1,11 @@
+{
+  "Info about this user": "",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/pt.json:2]:
```suggestion
{
```

File: backend/examples/user_custom_properties/default/locale/fr.json [hunk 15]
```diff
@@ -0,0 +1,11 @@
+{
+  "Info about this user": "Info sur l'utilisateur",
```
Comment (gbonaspetti) [inline backend/examples/user_custom_properties/default/locale/fr.json:2]:
```suggestion
{
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: backend/examples/user_custom_properties/default/locale/pt.json (hunk 14 above)
Comment (inkhey) [inline backend/examples/user_custom_properties/default/locale/pt.json:2]:
```suggestion
```

Comment (inkhey) [reply backend/examples/user_custom_properties/default/locale/pt.json]: done.

File: backend/examples/user_custom_properties/default/locale/fr.json (hunk 15 above)
Comment (inkhey) [inline backend/examples/user_custom_properties/default/locale/fr.json:2]:
```suggestion
```

**Review by gbonaspetti — APPROVED** [review]

File: backend/examples/user_custom_properties/default/locale/fr.json [hunk 16]
```diff
@@ -0,0 +1,10 @@
+{
+  "Tagline": "Slogan",
+  "Localization": "Localisation",
+  "Hobbies": "Loisir",
```
Comment (PhilippeAccorsi) [inline backend/examples/user_custom_properties/default/locale/fr.json:4]: Loisir => Loisirs

Comment (PhilippeAccorsi) [reply backend/examples/user_custom_properties/default/locale/fr.json:4]: But i'm not sure

File: backend/examples/user_custom_properties/default/locale/fr.json [hunk 17]
```diff
@@ -0,0 +1,10 @@
...
+  "Personal Website": "Site web personnel",
+  "Birthday": "Date d'anniversaire",
+  "Telephone": "Numéro de téléphone",
```
Comment (PhilippeAccorsi) [inline backend/examples/user_custom_properties/default/locale/fr.json:7]: `Telephone` is not `Phone` or `Phone number` ?

File: backend/examples/user_custom_properties/default/locale/pt.json [hunk 18]
```diff
@@ -0,0 +1,10 @@
+{
+  "Tagline": "Slogan",
+  "Localization": "Localização",
+  "Hobbies": "Passatempos",
+  "Personal Website": "Website pessoal",
+  "Birthday": "Data de nascimento",
+  "Telephone": "Telefone",
```
Comment (PhilippeAccorsi) [inline backend/examples/user_custom_properties/default/locale/pt.json:7]: Is it `Telephone` or `Phone` or `Phone number` ?

File: backend/examples/user_custom_properties/default/schema.json [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+      "items": {
+        "type": "string"
+      }
+    },
+    "personal_page": {
+      "type": "string",
+      "title": "Personal Page"
+    }
+  }
+}
```
Comment (PhilippeAccorsi) [inline backend/examples/user_custom_properties/default/schema.json:49]: Why we not use uppercase and not lowercase (Sometime we seen some uppercase

File: backend/examples/user_custom_properties/default/ui.json [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+  "hobbies":{
+    "tracim:display_group": "public_profile_first"
+  },
+  "website":{
+    "tracim:display_group": "public_profile_first"
+  },
+  "birthday": {
+    "tracim:display_group": "public_profile_first"
+  },
+  "telephone": {
```
Comment (PhilippeAccorsi) [inline backend/examples/user_custom_properties/default/ui.json:17]: Sme question here about Phone or Phone number

**Review by PhilippeAccorsi — CHANGES_REQUESTED** [review]
General question : why we not use uppercase for the first letter.

File: backend/examples/user_custom_properties/default/locale/fr.json (hunk 16 above)
Comment (inkhey) [reply backend/examples/user_custom_properties/default/locale/fr.json:4]:
```suggestion
  "Hobbies": "Loisirs",
```

File: backend/examples/user_custom_properties/default/locale/fr.json (hunk 17 above)
Comment (raphj) [reply backend/examples/user_custom_properties/default/locale/fr.json:7]: Phone or Phone number indeed.
Phone is shorter.

File: backend/examples/user_custom_properties/default/schema.json (hunk 19 above)
Comment (inkhey) [reply backend/examples/user_custom_properties/default/schema.json:49]: parameter name vs value to show in ui (title).

File: backend/examples/user_custom_properties/default/locale/fr.json (hunk 17 above)
Comment (inkhey) [reply backend/examples/user_custom_properties/default/locale/fr.json:7]:
```suggestion
  "Phone": "Numéro de téléphone",
```

File: backend/examples/user_custom_properties/default/schema.json (hunk 19 above)
Comment (raphj) [reply backend/examples/user_custom_properties/default/schema.json:49]: What do you mean?

File: backend/examples/user_custom_properties/default/locale/pt.json (hunk 18 above)
Comment (inkhey) [reply backend/examples/user_custom_properties/default/locale/pt.json:7]:
```suggestion
  "Phone": "Telefone",
```

File: backend/examples/user_custom_properties/default/ui.json (hunk 20 above)
Comment (raphj) [reply backend/examples/user_custom_properties/default/ui.json:17]: `phone`

File: backend/examples/user_custom_properties/default/schema.json [hunk 21]
```diff
@@ -0,0 +1,49 @@
...
+    },
+    "birthday": {
+      "type": "string",
+      "format": "date",
+      "title": "Birthday"
+    },
+    "telephone": {
+      "type": "string",
+      "title": "Telephone",
+      "minLength": 10
```
Comment (inkhey) [inline backend/examples/user_custom_properties/default/schema.json:35]:
```suggestion
    "phone": {
      "type": "string",
      "title": "Phone",
      "minLength": 10
```

File: backend/examples/user_custom_properties/default/schema.json (hunk 19 above)
Comment (inkhey) [reply backend/examples/user_custom_properties/default/schema.json:49]: the value have first character in uppercase whereas for title whereas the parameter are all lowercase

File: backend/examples/user_custom_properties/default/ui.json (hunk 20 above)
Comment (inkhey) [inline backend/examples/user_custom_properties/default/ui.json:17]:
```suggestion
  "phone": {
```

Comment (inkhey) [reply backend/examples/user_custom_properties/default/ui.json]: done.

**Review by PhilippeAccorsi — APPROVED** [review]

File: backend/examples/user_custom_properties/default/schema.json (hunk 19 above)
Comment (PhilippeAccorsi) [reply backend/examples/user_custom_properties/default/schema.json]: Ok for the answer

**Pull request merged**